src/mmw/js/src/modeling/gwlfeModificationConfig.js

```diff
@@ -224,6 +230,30 @@ function makeCurveAdjustingAgBmpConfig(outputName, tillFactor) {
     };
 }
 
+function makeCropTillageBmpConfig(outputName, tillFactor, efficiencies) {
+    function getOutput(inputVal, fractionVal, dataModel) {
+        var currentCN = dataModel[CurveNumberName][CroplandIndex],
+            newCN = adjustCurveNumber(currentCN, fractionVal, tillFactor || 1),
```

Comment:
In case this matters for the calculations, I noticed that the `tillFactor` argument will be default to 1 if `tillFactor` is `undefined`, `null`, or `0` since they're all falsy. The first two cases seem right to me, but I wanted to check if it should be set to 1 if it has explicitly been set to `0`.

```node
> function hello(arg) {
... console.log(arg);
... }
undefined
> hello(0 || 1)
1
undefined
```